amoro-ams/amoro-ams-server/src/main/resources/mysql/upgrade-0.7.0-to-0.7.1.sql

```diff
@@ -0,0 +1,39 @@
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+RENAME TABLE table_runtime TO table_runtime_backup;
+CREATE TABLE table_runtime LIKE table_runtime_backup;
+
+ALTER TABLE table_runtime CHANGE COLUMN optimizing_status optimizing_status_code INT DEFAULT 7;
+CREATE INDEX idx_optimizer_status_and_time ON table_runtime(optimizing_status_code, optimizing_status_start_time DESC);
+
+INSERT INTO table_runtime(
```

Comment:
For a table, is it necessary to back up and rebuild the table when adding new fields and converting strings to integers according to the rules?
Executing an UPDATE SQL should be sufficient.

Reply:
we'll backup `table_runtime` into `table_runtime_backup` and  insert the transformed value(string -> int) from `table_runtime_backup`, if there is some problem, we can reconstruct from the backup table. IMHO, this is enough for migration.